src/transactions/test/InvokeHostFunctionTests.cpp

```diff
@@ -402,12 +402,23 @@ TEST_CASE("basic contract invocation", "[tx][soroban]")
                       SorobanInvocationSpec const& spec,
                       std::optional<uint32_t> expectedRefund = std::nullopt,
                       bool addContractKeys = true) {
-        int64_t initBalance = test.getRoot().getBalance();
+        auto rootAccount = test.getRoot();
```

Comment:
Note: I've added a new TestDBMode so you don't have to do this manually, I'll update in my branch after rebasing.